hnn_core/dipole.py

```diff
@@ -106,25 +105,20 @@ def simulate_dipole(net, tstop, dt=0.025, n_trials=None, record_vsec=False,
     return dpls
 
 
-def _read_dipole_txt(fname, extension='.txt'):
+def _read_dipole_txt(fname):
```

Comment:
@katduecker here is the problem! Since the time you had copied `dipole.py` to your desktop the master branch has changed. Therefore these modifications are undoing the latest commits to the master branch.

I would reccomend copying this function exactly as it's written on the current master branch and replacing it in your code. That way the only changes that show up in the diff are related to the `tstop` PR.

If done correctly you will no longer see `_read_dipole_txt()` show up in the files changed.